api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -151,26 +151,24 @@ User user() {
 
   @Before
   public void setUp() {
-    WorkbenchConfig config = new WorkbenchConfig();
+    config = new WorkbenchConfig();
+    config.server = new WorkbenchConfig.ServerConfig();
+    config.server.apiBaseUrl = API_BASE_URL;
     config.firecloud = new WorkbenchConfig.FireCloudConfig();
     config.firecloud.registeredDomainName = "";
     config.access = new WorkbenchConfig.AccessConfig();
     config.access.enableComplianceTraining = true;
-    User user = new User();
+    config.featureFlags = new FeatureFlagsConfig();
+    config.featureFlags.useBillingProjectBuffer = false;
+
+    user = new User();
     user.setEmail(LOGGED_IN_USER_EMAIL);
     user.setUserId(123L);
     user.setFreeTierBillingProjectName(BILLING_PROJECT_ID);
     user.setFreeTierBillingProjectStatusEnum(BillingProjectStatus.READY);
-    when(userProvider.get()).thenReturn(user);
-    clusterController.setUserProvider(userProvider);
 
     createUser(OTHER_USER_EMAIL);
 
-    UserService userService = new UserService(
-        userProvider, userDao, adminActionHistoryDao, CLOCK, new FakeLongRandom(123),
-        fireCloudService, Providers.of(config), complianceService, directoryService);
-    clusterController.setUserService(userService);
-
```

Comment:
should the `testFcCluster` and `testCluster` have their names changed to match the new pattern?

Reply:
Good catch. Since these tests have the flag off by default, I think we just leave it as-is for now, but added a TODO to update these later. The name here is actually not super pertinent to the tests so I don't think it's awful to have some temporary inconsistency here.